stream-chat-android-client/src/main/java/io/getstream/chat/android/client/uploader/ProgressTracker.kt

```diff
@@ -0,0 +1,41 @@
+package io.getstream.chat.android.client.uploader
```

Comment:
`@InternalStreamChatApi` for everything in this file as well?

Reply:
As we are discussing using DiffUtils instead if ProgressTracker, let's move it to `stream-chat-android-ui-components` and make then `internal`. Although they are not a UI component, I believe it is better keep then there so we can improve the communication with the views in a next PR and don't expose those classes that are going away. What do you think?

Reply:
If we don't depend on this or the factory in the client module, then yes, moving them to the UI components module makes sense.

Reply:
Yeah... not possible... 2 different modules depend of this class. Let's put the annotation then.